internal/models/cluster/integration/search_scope.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package integration
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// VmwareTanzuManageV1alpha1ClusterIntegrationSearchScope Scope to search by, any fields left empty will be considered all (*).
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.integration.SearchScope
+type VmwareTanzuManageV1alpha1ClusterIntegrationSearchScope struct {
```

Comment:
If the search scope is not used, can we please remove the struct ?

Is list api being invoked ?

Reply:
No list API - removing related models.